app/assets/javascripts/discourse/tests/integration/components/user-menu/likes-notifications-list-test.js

```diff
@@ -0,0 +1,44 @@
+import { module, test } from "qunit";
+import { setupRenderingTest } from "discourse/tests/helpers/component-test";
+import { render } from "@ember/test-helpers";
+import { cloneJSON } from "discourse-common/lib/object";
+import NotificationFixtures from "discourse/tests/fixtures/notification-fixtures";
+import { hbs } from "ember-cli-htmlbars";
+import pretender from "discourse/tests/helpers/create-pretender";
+
+function getNotificationsData() {
+  return cloneJSON(NotificationFixtures["/notifications"].notifications);
+}
+
+module(
+  "Integration | Component | user-menu | likes-notifications-list",
+  function (hooks) {
+    setupRenderingTest(hooks);
+
+    let queryParams = null;
+    hooks.beforeEach(() => {
+      pretender.get("/notifications", (request) => {
+        queryParams = request.queryParams;
+        return [
+          200,
+          { "Content-Type": "application/json" },
+          { notifications: getNotificationsData() },
+        ];
+      });
+    });
+
+    hooks.afterEach(() => {
+      queryParams = null;
+    });
+
+    const template = hbs`<UserMenu::LikesNotificationsList/>`;
+
+    test("requests notifications filtered by the `liked` and `liked_consolidated` types", async function (assert) {
+      await render(template);
+      assert.strictEqual(
+        queryParams.filter_by_types,
```

Comment:
I think we need to review this type of tests which are already covered by a high level integration test in `user-menu/menu-test.`

Reply:
removed this and `replies-notification-list-test.js`.